src/core/plugin/Plugin.h

```diff
@@ -133,6 +138,10 @@ class Plugin final {
     bool valid = false;           ///< Flag if the plugin is valid / correct loaded
 };
 
+constexpr bool operator==(Plugin const& lhs, Plugin const& rhs) noexcept { return tieer(lhs) == tieer(rhs); }
+
+constexpr bool operator<(Plugin const& lhs, Plugin const& rhs) noexcept { return tieer(lhs) == tieer(rhs); }
```

Comment:
I forgot, that the comparison is by dept first. But does it matter in this case?

Reply:
In particular, I think this can be fixed with
```suggestion
constexpr bool operator<(Plugin const& lhs, Plugin const& rhs) noexcept { return tieer(lhs) < tieer(rhs); }
```
Otherwise sorting will not work correctly.